src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+```python
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# Define the path where the Parquet files are located
+data_path = '/home/eduardo_1999/projects/zrive-ds/projects/zrive-ds/sampled-datasets/'
+
+# Load the Parquet files using the full path
+try:
+    orders = pd.read_parquet(data_path + 'orders.parquet')
+    regulars = pd.read_parquet(data_path + 'regulars.parquet')
+    abandoned_cart = pd.read_parquet(data_path + 'abandoned_carts.parquet')  # Ensure to use the correct name
+    inventory = pd.read_parquet(data_path + 'inventory.parquet')
+    users = pd.read_parquet(data_path + 'users.parquet')
+except Exception as e:
```

Comment:
This try - excepts is making more harm than good. 
Look, with try-except:

- Extra complexity
- The cell does NOT fail --> the notebook continues executing and will fail later on with a worse error code